installer/data/template/.origintrail_noderc_one_click_mainnet.json

```diff
@@ -5,34 +5,53 @@
         "otp:2043":{
           "enabled":true,
```

Comment:
Not sure if this should also be parametrised, if someone doesn't want to run node on some network